star-dash/star-dash/GameEngine/Systems/Complex/DeathSystem.swift

```diff
@@ -83,9 +83,12 @@ class DeathSystem: System {
             return
         }
 
+        if let jumpComponent = entityManager.component(ofType: JumpComponent.self, of: event.playerId) {
```

Comment:
Should the responsibility of removing jumpComponent be from the JumpModule in MovementSystem instead? Or maybe we could do something like MovementSystem.cancelAllMovement(for: entityId)

Reply:
ok. moved logic to MovementSystem. DeathSystem now calls .cancelAllMovement(for: entityId)